### ISSUE TITLE
KiCAD 3D View is broken

### ISSUE THREAD

**Issue by shreekumar3d** [balub/CoryDora#3]
CoryDora.step looks complete.  However, the 3D view in KiCAD is broken - it doesn't show 3D models when I clone.

Looks like this is due to local references.  In the footprint properties, I see 3D models referenced such as 
- ${SCOTTOKEEBS_KICAD}/3dmodels/ScottoKeebs_Hotswap.3dshapes/Hotswap_MX.step
- ${SCOTTOKEEBS_KICAD}/3dmodels/ScottoKeebs_Components.3dshapes/Diode_DO-35.step

Where can I find these source files ?

Two ways to fix this:

1. Provide instructions on where to get the models and how to use them
2. Make them part of the project

**Comment by balub** [comment]
Yes @shreekumar3d sir, I intent to add those files as sub-modules to the repo.

**Comment by shreekumar3d** [comment]
Sure @balub . Can you say where those models come from ? I will quickly try to pull these in, while you do the sub-modules stuff.
